gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```

Comment:
Is this Gradle upgrade required to make this work with the `osdetector` plugin? If not, I would probably send it as a separate PR.

Reply:
I reverted the Gradle upgrade in this PR, will send a separate PR.